src/fp.c

```diff
@@ -95,13 +95,13 @@ void fpx_init(struct fpx **fpx_handle) {
  * \return none
  */
 void fpx_update (struct fpx *fpx, 
-		    const struct pcap_pkthdr *header, 
-		    const void *data, 
-		    unsigned int len, 
-		    unsigned int report_fpx) {
+                 const struct pcap_pkthdr *header, 
+                 const void *data, 
+                 unsigned int len, 
+                 unsigned int report_fpx) {
     struct extractor x;
```

Comment:
This is a sample issue report.  ![facebook-smiley-face-for-comments](https://user-images.githubusercontent.com/5769903/60592930-a532a080-9d6f-11e9-9975-61fb18b67331.png)

Click to share feedback:
[Helpful!](http://muse.dev/thanks)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[Not Useful.](http://muse.dev/thanks)